src/index.html

```diff
@@ -8,6 +8,12 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
+      <div class="star--1"></div>
+      <div class="star--2"></div>
+      <div class="star--3"></div>
+      <div class="star--4"></div>
+      <div class="star--5"></div>
+    </div>
```

Comment:
now, these divs should also have the `stars` class. also, put five stars with class `stars__star` inside each div